### PR TITLE
Removed commented-out code

### PR DESCRIPTION
**Pull request by CrazyPython**
what's it doing there?

### CODE REVIEW

**Comment by CrazyPython** [comment]
@casperdcl update?

**Comment by CrazyPython** [comment]
How is codecov *still* not done yet?

Also, why's that commented out code there, @casperdcl?

File: tqdm/_tqdm_gui.py [hunk 1]
```diff
@@ -38,13 +38,10 @@ def write(cls, s, file=sys.stdout, end="\n"):
 
     def __init__(self, *args, **kwargs):
 
-        # try:  # pragma: no cover
+        # pragma: no cover
```
Comment (casperdcl) [inline tqdm/_tqdm_gui.py]: can you remove this one line (41) too?

Comment (CrazyPython) [inline tqdm/_tqdm_gui.py]: @casperdcl okay, it will affect coverage though

**Comment by CrazyPython** [comment]
@casperdcl also removed an empty line

**Pull request merged**